packages/story-editor/src/components/hotlinkModal/useHotlinkModal.js

```diff
@@ -107,6 +109,41 @@ function useHotlinkModal({
         return;
       }
 
+      if (
+        'image' === hotlinkInfo?.type &&
+        requiredImgDimensions?.height &&
+        requiredImgDimensions?.width
+      ) {
+        const proxiedUrl = needsProxy
+          ? getProxiedUrl({ needsProxy }, link)
+          : link;
+
+        const dimensions = await getImageDimensions(proxiedUrl);
```

Comment:
We need to restore parts of 6fd63a28493a98e504b0431ce016e6270d07d24a again. These should stay.

We need different messages if:

1. Only required height is provided
2. Only required width is provided
3. Both are provided

Reply:
There are **NO** cases where height and width are not provider. So I didn't see the point of writing for an edge case that will not happen. But I am okay to add this for the future.

Reply:
Fixed.

Reply:
> There are NO cases where height and width are not provider. So I didn't see the point of writing for an edge case that will not happen. 

But then the duplicate separate checks for `requiredImgDimensions?.height`and `requiredImgDimensions?.width` don't make sense.